File: Data Structures & Algorithms/longest-consecutive-sequence/submission-4.java
class Solution {
    public int longestConsecutive(int[] arr) {
        Arrays.sort(arr);
        int count = 1;
        int n = arr.length;
        int res = Integer.MIN_VALUE;
        for(int i=1;i<n;i++){

            if(arr[i]==arr[i-1]+1){
                count++;
            }
            else if(arr[i]==arr[i-1]){
                continue;
            }
            else{
                count= 1;
            }


            res = Math.max(res,count);
        }

        return res;
    }
}

File: Data Structures & Algorithms/can-place-flowers/submission-4.py
class Solution:
    def canPlaceFlowers(self, arr: List[int], n: int) -> bool:
        count = 0
        for i in range(len(arr)):
            if i==0 and arr[i+1]==0 and arr[i]==0:
                count+=1
                arr[i]=1
            elif i==len(arr)-1 and arr[len(arr)-1]==0 and arr[len(arr)-2]==0:
                arr[i]=1
                count+=1
            elif arr[i]==0 and arr[i+1]==0 and arr[i-1]==0:
                arr[i]=1
                count+=1
        
        return count>=n

